sqlgen/sql_dialect_options.go

```diff
@@ -239,7 +241,7 @@ type (
 		// The order of SQL fragments when creating a SELECT statement
 		// (Default=[]SQLFragmentType{
 		// 		CommonTableSQLFragment,
-		// 		SelectSQLFragment,
+		// 		SelectSQLFragment or SelectWithLimitSQLFragment,
```

Comment:
This should be reverted, since the default has not changed.

Reply:
fixed